src/reanimated2/index.ts

```diff
@@ -208,37 +211,34 @@ export {
   SharedTransition,
   SharedTransitionType,
 } from './layoutReanimation';
-export { getRelativeCoords, ComponentCoords, isSharedValue } from './utils';
-export {
+export type { ComponentCoords } from './utils';
+export { getRelativeCoords, isSharedValue } from './utils';
+export type {
   StyleProps,
   SharedValue,
   AnimatableValueObject,
   AnimatableValue,
   AnimationObject,
-  Animation,
-  SensorType,
-  IOSReferenceFrame,
   SensorConfig,
+  Animation,
   AnimatedSensor,
   AnimationCallback,
   Value3D,
   ValueRotation,
-  InterfaceOrientation,
-  KeyboardState,
   AnimatedKeyboardInfo,
-  MeasuredDimensions,
   AnimatedKeyboardOptions,
+  MeasuredDimensions,
+} from './commonTypes';
+export {
+  SensorType,
+  IOSReferenceFrame,
+  InterfaceOrientation,
+  KeyboardState,
   ReduceMotion,
 } from './commonTypes';
 export { FrameInfo } from './frameCallback';
 export { getUseOfValueInStyleWarning } from './pluginUtils';
-export {
-  withReanimatedTimer,
-  advanceAnimationByTime,
-  advanceAnimationByFrame,
-  setUpTests,
-  getAnimatedStyle,
-} from './jestUtils';
+export * from './jestUtils';
```

Comment:
jestUtils is unused on web and has a mock, so by directly exporting these methods the build was throwing warnings. I checked and these are the exact exports of jestUtils so it is safe to export all. If this is an issue then we would need to create a copy of this file as a web override that can have no exports from jestUtils.